src/recipe.cpp

```diff
@@ -160,6 +170,14 @@ void recipe::load( const JsonObject &jo, const std::string &src )
         }
     }
 
+    if( jo.has_array( "proficiencies" ) ) {
+        for( const JsonObject &jsobj : jo.get_array( "proficiencies" ) ) {
+            recipe_proficiency nprof;
+            nprof.load( jsobj );
+            proficiencies.emplace_back( nprof );
+        }
```

Comment:
```suggestion
        jo.read( "proficiencies", proficiencies );
```
(you may need to write `recipe_proficiency::deserialize` instead of `load` for this to work)

Reply:
Thanks, I'll fix that!